autoload/elixir/indent.vim

```diff
@@ -355,6 +369,39 @@ function! s:do_handle_pattern_match_block(relative_line, context)
   end
 endfunction
 
+function! elixir#indent#handle_inside_embedded_view(context)
+  if s:in_embedded_view()
```

Comment:
We might be able to simplify this with a syntax check, e.g.

Given

```elixir
def render(assigns) do
  ~H"""
  <Component
    id="some-id"
    value={{ @some_value}}
  >
    Inner HTML
  </Component>
  """
end
```

With `synstack(3, 3)` (note: cursor pos) we get: `[145, 396, 163, 157]`. Those can be mapped to the respective syntax group. `synIDattr(396, "name")` returns `elixirSurfaceSigil`. So we can dump the stack and search if we're nested in an embedded view this way